package.json

```diff
@@ -86,7 +86,6 @@
     "babel-plugin-transform-require-context": "0.1.1",
     "bn.js": "5.1.3",
     "browserslist": "4.8.6",
-    "chroma-js": "2.1.0",
```

Comment:
@pbearne Chroma JS is being used in `plugin/assets/src/customizer/components/material-library/index.js` Should we update the reference for it?